mechanics/RELEASE-SCHEDULE.md

```diff
@@ -5,25 +5,33 @@ weight: 50
 type: "docs"
 ---
 
-Knative releases every 6 weeks. As much as possible, releases should be driven by automation, and repos should be ready to release at any point. It should also be possible to produce and consume nightly artifacts.
+Knative releases quarterly. As much as possible, releases should be driven by automation, and repos should be ready to release at any point. It should also be possible to produce and consume nightly artifacts.
 
-With that said, it can be useful to have a list of when future releases will happen, so this document provides a schedule for the next 6+ months of releases.
+With that said, it can be useful to have a list of when future releases will happen, so this document provides a schedule for the next 12 months of releases.
+
+## Upcoming releases
 
 | Release | Date       | EOL        | Min K8s Version | Notes                         |
 | ------- | ---------- | ---------- | --------------- | ----------------------------- |
-| 1.8     | 2022-10-04 | 2023-04-04 | 1.23            | |
-| 1.9     | 2022-11-15 | 2023-05-16 | 1.23            | |
-| 1.10    | 2023-01-10 | 2023-06-27 | 1.24            | |
-
+| 1.8     | 2022-10-18 | 2023-05-02 | 1.23            | |
+| 1.9     | 2023-01-24 | 2023-08-01 | 1.24            | |
+| 1.10    | 2023-04-25 | 2023-10-31 | 1.25            | |
+| 1.11    | 2023-07-25 | 2024-01-30 | TBD             | |
+| 1.12    | 2023-10-24 | 2024-04-30 | TBD            | |
 
-## Historical (no longer supported) releases:
+## Releases supported by community
```

Comment:
Thanks for splitting this out; the other option would be to remove the `(no longer supported)` header, so that we don't need to move things between sections as often, but this works for me as well.